tests/resize_wrapper.F90

```diff
@@ -0,0 +1,29 @@
+!Copyright 2023 Meteo-France, ECMWF 
+!
+!Licensed under the Apache License, Version 2.0 (the "License");
+!you may not use this file except in compliance with the License.
+!You may obtain a copy of the License at
+!
+!    http://www.apache.org/licenses/LICENSE-2.0
+!
+!    Unless required by applicable law or agreed to in writing, software
+!    distributed under the License is distributed on an "AS IS" BASIS,
+!    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+!    See the License for the specific language governing permissions and
+!    limitations under the License.
+
+PROGRAM RESIZE_WRAPPER
+        !TEST IF RESIZING A WRAPPER IS FAILING AS IT SHOULD
+
+        USE FIELD_MODULE
+        USE FIELD_FACTORY_MODULE
+        USE PARKIND1
+        IMPLICIT NONE
+        CLASS(FIELD_2RB), POINTER :: W => NULL()
+        REAL(KIND=JPRB) :: D(5,5)
+
+        CALL FIELD_NEW(W, DATA=D)
+        CALL FIELD_RESIZE(W, UBOUNDS=[100,100])
```

Comment:
[just nitpicking] I take it the by marking this in the ABOR1_TEST_FILES failure gets checked? It could be helpful to make the exact place in code where that's supposed to happen.

Reply:
Added a comment to show the line calling abor1